### ISSUE TITLE
⌨ Accessibility - Profile update toaster message unclear 

### ISSUE THREAD

**Issue by RM-1978** [GCTC-NTGC/gc-digital-talent#5420]
## ❌ Current behaviour (issue)
When returning to the Application Profile page from a form (e.g., About Me, Language Information [language proficiency]), it is not clear from the toaster that the user profile was updated with the form information they entered. 
### AT configuration: 
- VoiceOver, IOS, Safari, 
- JAWS, Windows, Chrome/Edge
- NVDA, Windows, Chrome/Edge 
### How to reproduce 
1. Navigate using a screen reader and your keyboard to profile. 
2. Fill in/update the About info. 
3. Tap the Save and go back link
4. Read notification and confirm that you have updated application profile with the information you just entered. 
### Screen capture
![#1234 Screen Capture.JPG](https://images.zenhubusercontent.com/625076b8f67c446dd188ffe5/403469c0-3e52-42fd-9dd9-4fec3c17bbb7)
## ✔ Expected behaviour (fix)
Copy in the toaster message tells the user that their profile has been updated with the information they have just completed for all form sections. The way it is for Skills and Experience updates. 

![image.png](https://images.zenhubusercontent.com/625076b8f67c446dd188ffe5/a32758e7-2c58-4631-be5d-8e8c5c699b4a)
![#1234 Screen Capture.JPG]
### Acceptance Criteria
- [ ] Toaster notification tells the user that their profile has been updated with the information they have just completed. 
## Additional information (optional)
1. How does it impact users? Minor adjustment that gives users assurance that they have updated and saved the right information.

**Comment by esizer** [comment]
> @esizer @mnigh Do y'all need us to provide you with contextual copy for each situation something is updated, or is

That would be great! 😄 

> > Your [specific data] was successfully updated!
> 
> enough to standardize it?

This looks god to me 👍

**Comment by mnigh** [comment]
> @esizer @mnigh Do y'all need us to provide you with contextual copy for each situation something is updated, or is
> 
> > Your [specific data] was successfully updated!
> 
> enough to standardize it?

@substrae As long as this copy will work in French for different gender nouns and plurals.

**Comment by substrae** [comment]
I would probably need @gobyrne to confirm that for you @mnigh :)

**Comment by gobyrne** [comment]
We just need the name of the subpage that was saved in the toast message. "About me - updated successfully" would work.

**Comment by mnigh** [comment]
@gobyrne What about the _"Message displayed to user after user fails to get updated"_ that doesn't follow the same suggested format as the _"Message displayed to user after user is updated successfully"_ 
`${subpage} - updated successfully`, what should that look like so it will work in French as well?

https://github.com/GCTC-NTGC/gc-digital-talent/blob/aeeee8b069cea1818bdfde89b52dbd46532d2ab2/apps/web/src/pages/Profile/AboutMePage/components/AboutMeForm/AboutMeForm.tsx#L172

https://github.com/GCTC-NTGC/gc-digital-talent/blob/aeeee8b069cea1818bdfde89b52dbd46532d2ab2/apps/web/src/messages/profileMessages.ts#L10-L14

https://github.com/GCTC-NTGC/gc-digital-talent/blob/aeeee8b069cea1818bdfde89b52dbd46532d2ab2/apps/web/src/lang/fr.json#L968-L971

**Error: updating user failed**

**Comment by esizer** [comment]
I think we completed this when @JamesHuf updated the profile page to use inline forms. I just checked and each section seems to be showing contextual messages.

![Screenshot 2023-07-18 084757.png](https://images.zenhubusercontent.com/624af7203bd4eddbbafed4b8/40f1877f-292f-4335-abce-b49829158096)
